go/enclave/txpool/txpool.go

```diff
@@ -78,6 +79,14 @@ func (t *TxPool) Add(transaction *common.L2Tx) error {
 	return nil
 }
 
+//go:linkname validateTxBasics github.com/ethereum/go-ethereum/core/txpool/legacypool.(*LegacyPool).validateTxBasics
+func validateTxBasics(_ *legacypool.LegacyPool, _ *types.Transaction, _ bool) error
+
+// Validate - run the underlying tx pool validation logic
+func (t *TxPool) Validate(tx *common.L2Tx) error {
+	return validateTxBasics(t.legacyPool, tx, false)
```

Comment:
this will be hacky no matter how we do it.
At least this is simple

Reply:
Yeah fair enough. Not seen the `go:linkname` incantation before haha, that's a fun one.